src/components/ServiceRequestForm.jsx

```diff
@@ -17,7 +16,7 @@ import OtherAnimalsTypes from "./otherAnimalTypes";
 import SexType from './sexType';
 import AnimalColorType from './animalColorType';
 import AnimalBreedType from './animalBreedType';
-import { URLRouting } from '../utilities/FormHelpers';
+import { URLRouting, setFieldValues } from '../utilities/FormHelpers';
```

Comment:
We don't have a standards document so i can't block for this, but in these standalone js files the expecation is the function name is uppercase `SetFieldNames`. That way we know its coming from somewhere else.

Reply:
Changed and pushed